src/puppetlabs/trapperkeeper/services/authorization/authorization_core.clj

```diff
@@ -56,6 +56,13 @@
   (->> (select-keys config-map #{:allow :allow-ip :deny :deny-ip})
        (reduce #(add-individual-acl (first %2) (second %2) %1) rule)))
 
+(defn add-query-params
+  "Add any query parameters specified in configuration to the rule."
+  [rule {:keys [query-params]}]
+  (if query-params
```

Comment:
Maybe. The `query-params` here is straight from the config, so it's possible it will be `nil`.  I think the `rules/query-param` will gladly add empty/nil values, which we don't want, so I put the conditional here.

I'll play around with it a bit to see if we can improve it.